components/badge/badge_pins.h

```diff
@@ -140,6 +140,40 @@
 #define MPR121_PIN_NUM_LEDS     10
 #define MPR121_PIN_NUM_SD_CD    11
 
+#elif defined(CONFIG_SHA_BADGE_V3_LITE)
```

Comment:
Because we see it as a different board at the moment, I think code duplication isn't really a problem. To ease some things, we could split the badge_pins.h in a badge_config_v1.h, badge_config_v2.h, badge_config_v3.h and badge_config_v3_lite.h. Then we can use diff to spot the differences between these files.